### PR TITLE
[8.0][IMP][mrp_project] Set workorder in analytic.line related with a task from a work order

### PR DESCRIPTION
**Pull request by antespi**
This PR improves @ddico work added here https://github.com/OCA/manufacture/commit/b82abb6ecea258586e34802d48e9a700d935118b
Setting also work order to `account.analytic.line` when creating task work from a task related with a work order

### CODE REVIEW

File: mrp_project/__openerp__.py [hunk 1]
```diff
@@ -6,10 +6,11 @@
 {
     "name": "MRP Project Link",
     "summary": "Link production with projects",
-    "version": "8.0.1.1.0",
+    "version": "8.0.1.1.1",
     "depends": [
         "mrp_analytic",
         "project",
+        "project_timesheet",
```
Comment (pedrobaeza) [inline mrp_project/__openerp__.py:13]: Why this new dependency? This breaks current installations.

Comment (pedrobaeza) [inline mrp_project/__openerp__.py:13]: A new addon _mrp_project_timesheet_?

Comment (antespi) [inline mrp_project/__openerp__.py:13]: This dependency is necessary for enabling inheritance in this method: https://github.com/OCA/manufacture/pull/94/files#diff-8655ff19eb2fb4b321a2742bdcf95e8eR33 `_create_analytic_entries`

Comment (pedrobaeza) [inline mrp_project/__openerp__.py:13]: OK, go on with it. You're not going to find any installation that enables real costs without it, but please change version to 8.0.1.2.0

Comment (antespi) [inline mrp_project/__openerp__.py:13]: Done, thanks

**Comment by pedrobaeza** [comment]
:+1:

**Comment by rafaelbn** [comment]
:+1:

**Comment by sergio-teruel** [comment]
:+1:

**Pull request merged**